packages/moxb/src/table/TableSearchImpl.ts

```diff
@@ -34,14 +34,18 @@ export class TableSearchImpl implements TableSearch {
         enabled: () => this.searchField.value !== '',
     });
 
+    constructor() {
+        makeObservable(this);
+    }
+
     @action.bound
     setQuery(query: string) {
         this.query = query;
     }
 
     readonly queryFilter: QueryFilter = new QueryFilterImpl({
         getQuery: () => this.query,
-        setQuery: this.setQuery,
+        setQuery: (value) => this.setQuery(value),
```

Comment:
The problem is that it will only be bound when the `makeObservable(this)` call runs, in the constructor of this class.
However, the fields of the object instance are initiated _before_ that.  Here, we are initiating the `queryFilter` field... so when this code is executed, the constructor hasn't been executed yet, so the method is not _yet_ bound.

Hence, I need to do this.
Another way would be to initialize the fields inside the constructor... but then I would have to have a separate field declaration at one location of the code, and then assigning the value later (in the constructor).

A third option would be to try to fix this by modifying the TS config of the project (because I seem to remember that are options for configuring this behavior), but then our code would depend on a specific combination of TS config values, which would be fragile...